### ISSUE TITLE
Color codes don't render correctly in my terminal (zsh)

### ISSUE THREAD

**Issue by devth** [liquidz/vim-iced#107]
```bash
iced repl
\e[32mOK\e[m: \e[1mLeiningen project is detected\e[m
```

Screenshot:

![image](https://user-images.githubusercontent.com/86107/54868702-fedd0500-4d54-11e9-9708-83c151947d48.png)

**Comment by liquidz** [comment]
@devth Thanks you for reporting!
This may be the cause.
https://stackoverflow.com/questions/28782394/how-to-get-osx-shell-script-to-show-colors-in-echo

Could you try `feature/iced-command-color` branch? (73ac3817f8f90acd094cca8a6441edc9c52fe328)

**Comment by devth** [comment]
Looks good on `feature/iced-command-color`!!

![image](https://user-images.githubusercontent.com/86107/54883740-834f8680-4e2e-11e9-99dd-4152252383f0.png)

**Comment by liquidz** [comment]
Closed by #115
